.changeset/tricky-comics-know.md

```diff
@@ -0,0 +1,49 @@
+---
+'@xstate/store': minor
+---
+
+Added `createAtom()` for creating reactive atoms that can be combined with other atoms and stores:
+
+- Create simple atoms with initial values:
+
+  ```ts
+  import { createAtom } from '@xstate/store';
+
+  const countAtom = createAtom(0);
+  countAtom.get(); // 0
+  countAtom.set(1); // or use setter function: (prev) => prev + 1
+  ```
+
+- Subscribe to atom changes:
+
+  ```ts
+  countAtom.subscribe((value) => console.log(value));
+  ```
+
+- Combine multiple atoms:
+
+  ```ts
+  const nameAtom = createAtom('hello');
+  const countAtom = createAtom(3);
+  const combinedAtom = createAtom((read) =>
+    read(nameAtom).repeat(read(countAtom))
```

Comment:
That `repeat` threw me off at first 😅 i thought it was some magic new method but that’s just the unwrapped value’s method :D